src/main/java/com/mycompany/myapp/repository/ToolRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.mycompany.myapp.repository;
+
+import com.mycompany.myapp.domain.Pool;
+import com.mycompany.myapp.domain.Tool;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ToolRepository extends JpaRepository<Tool, String> {
+    List<Tool> findAllByPool(Pool pool);
```

Comment:
I am not entirely convinced that this works. Did you test it? Don't you need a custom query to make a "join" or a where clause?

Reply:
Yes I tested it, do you want special commit to test it also ?